migrations/2023-09-17-152010_make_id_not_null_address/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE address ALTER COLUMN id DROP NOT NULL;
```

Comment:
What about making the `address_id` field as primary key? I noticed that the code includes that change, but can't see the corresponding migration...

Reply:
`address_id` was already made as primary key during table creation